Setup-and-CLI.md

```diff
@@ -0,0 +1,92 @@
+# Setup and Common Commands
+
+> This document provides a collection of common command line options we used with GraalVM and Truffle on TruffleSqueak.
+
+## Setup GraalVM
+
+Follow the GraalVM installation guide [3] for step-by-step instructions on installing GraalVM. The installation guide provides instructions for different operating systems.
+
+## Setup TruffleSqueak
+
+Follow the TruffleSqueak setup guide [2] for step-by-step instructions on setting up the TruffleSqueak project. The setup guide provides instructions for different operating systems and IDEs.
+
+To build the TruffleSqueak executable, we use the `⁠mx` tool, which is part of the GraalVM ecosystem [1]. We call `mx` when the current working directory is the `trufflesqueak` directory with `../mx/mx`.
+
+## Export GraalVM Home
+
+Before running any commands, you need to set the `GRAALVM_HOME` environment variable to point to your GraalVM installation. You can do this by running the following command in your terminal:
+
+```bash
+export GRAALVM_HOME=$(mx --dy /compiler graalvm-home) 
+```
+
+## Build
+
+We use the `mx` tool to build the TruffleSqueak project. Before building the project, you need to ensure that `GRAALVM_HOME` is set correctly. The command below builds the TruffleSqueak.
+
+```bash
+mx --dy trufflesqueak,/compiler build
+```
+
+## Run
+
+To run an image, we use the built artifacts. You can run an image with the following command:
```

Comment:
Is there already a reference on how to get the following mentioned test-64bit.image? Also TruffleSqueak will automatically download a base image if no image is specified. Should we mention that? 🙂